contracts/other/notional/README.md

```diff
@@ -0,0 +1,35 @@
+# Notional fCash as collateral
+
+## About fCash
+Notional borrowing and lending positions are stored as [fCash](https://docs.notional.finance/notional-v2/notional-v2-basics/fcash).
+
+fCash positions can be positive or negative. If positive they are equivalent for a lending position, and are redeemable at maturity for their face value. If negative they are borrowing positions.
+
+We want to use fCash lending positions as collateral. Given that we know that they are redeemable for their face value at maturity and their volatility is low, they are good assets for that purpose.
+
+Accepting fCash as collateral also eases arbitraging and leverage between Notional an Yield. By taking positive fCash balances as collateral, we facilitate lending on Notional and borrowing from Yield.
+
+fCash is kept internally in the Notional contracts, but there is a [proxy](https://github.com/notional-finance/contracts-v2/blob/master/contracts/external/actions/ERC1155Action.sol) that allows working with fCash as an ERC1155 token.
+
+The Notional [proxy](0x1344A36A1B56144C3Bc62E7757377D288fDE0369) is deployed on the Ethereum mainnet. It has links to the [ERC1155](0xffd7531ed937f703b269815950cb75bdaaa341c9) and [Views](0xde14d5f07456c86f070c108a04ae2fafdbd2a939) proxies. Not sure but I think that you call all functions on the main proxy.
```

Comment:
Can we update this comment.  From what I saw we are only calling `safeTransferFrom` and when I go to the proxy address I don't see that fn exposed. https://etherscan.io/address/0x1344A36A1B56144C3Bc62E7757377D288fDE0369#writeProxyContract

Reply:
There are more proxies behind the main proxy, which expose that function.

![image](https://user-images.githubusercontent.com/38806121/155304847-91ea6756-2013-4d07-83a3-20a6868b8ead.png)

Reply:
Great.  Can we remove the "Not sure but I think..." from the README?

Reply:
Done. Also tested that yesterday.